ci/checks/changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
If you guys don't want to manually manage the changelog, I can add our `ops-bot` application to this repo. The `ops-bot` will automatically add entries to a GitHub pre-release (i.e. [this](https://github.com/rapidsai/cudf/releases/tag/v21.08.00a)) anytime a PR is merged. This might be best if we want this repo to align with the other RAPIDS repos.

Reply:
That would be perfect! I will remove the changelog.sh